.coveragerc

```diff
@@ -9,6 +9,8 @@ omit =
     invoicing_app/admin.py
     invoicing_app/models.py
     invoicing_app/management/commands/generate_tax_receipts_old.py
+    invoicing_app/management/commands/update_incomplete_tax_receipts.py
+    invoicing_app/management/commands/compare_generate.py
     invoicing_app/management/commands/create_dummy_data.py
```

Comment:
Why are we omitting these commands? Shouldn't be tested?

Reply:
The comparing script was intended as an internal tool to test output of both scripts. We haven't even decided if it is of any use to prod environment. The update tax receipt script was omitted because we will add tests next sprint, this was just the first approach.

Reply:
Okay then! You should not omit for the coverage tool, tho. If something will be tested in the future but it's not being tested today, that should be reflected in the coverage report